R/predict.R

```diff
@@ -199,3 +199,47 @@ check_fitted <- function(model_stack) {
     )
   }
 }
+
+#' @importFrom generics augment
+#' @export
+generics::augment
+
+#' Augment a model stack
+#' 
+#' @param x A fitted model stack; see [fit_members()].
+#' @inheritParams predict.model_stack
+#' @param ... Additional arguments passed to `predict.model_stack`. In
+#' particular, see `type` and `members`.
+#' 
+#' @seealso The [collect_parameters()] function is analogous to a [`tidy()`][generics::tidy()] 
+#' method for model stacks.
+#' 
+#' @method augment model_stack
+#' @name augment.model_stack
+#' @export
+augment.model_stack <- function(x, new_data, ...) {
+  dots <- list(...)
+  outcome <- x[["outcome"]]
+  member_cols <- unlist(x[["cols_map"]])
+  
+  res <- dplyr::bind_cols(new_data, predict(x, new_data = new_data, ...))
+  
+  if (isTRUE(dots[["members"]]) & 
```

Comment:
This condition feels quite long and similar to the next one. Maybe refactor it out so it can be reused

Reply:
Sure thing, added in [9127553](https://github.com/tidymodels/stacks/pull/176/commits/9127553d2efa567b9bea04ac038f9ea73e7406db)!